myeasylog.log

```diff
@@ -0,0 +1,24 @@
+2022-04-04 07:29:31,503 WARNING [default] No input received when trying to open json from file game1.json. Does the file exist?
```

Comment:
Going forward, we can each probably delete logs and .exe before creating PR's from now on, they just create unnecessary conflicts.

Reply:
In my PR I have created a .gitignore file so the logs/ folder and .vscode/ folder can be ignored

Reply:
@studioant @Delcari Ahh, okay thankyou, I will create this .gitignore file and add these to it.